### ISSUE TITLE
createObject() function not recognized by Azure

### ISSUE THREAD

**Issue by kamilzzz** [microsoft/vscode-azurearmtools#1043]
Hi,

From latest release:
`Support for new template functions: createObject(), false(), true(), null()`

Where is createObject() ARM template function documented? I couldn't find it anywhere and when I try to deploy template using createObject() function it fails with `'The template function 'createObject' is not valid. Please see https://aka.ms/arm-template-expressions for usage details.'`.

Schema version used by template: `"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",`

**Comment by StephenWeatherford** [comment]
@kamilzzz Sorry, should have said in changelog - these were preemptively added, they're not actually available publicly in Azure yet.

**Comment by StephenWeatherford** [comment]
@bmoore-msft When are these publicly available?  It looks like from comment in #971 that they should already be there.

**Comment by bmoore-msft** [comment]
They should be there now, finished deploying last week.

**Comment by StephenWeatherford** [comment]
@kamilzzz Is this the main Azure cloud, or is it a country-specific one or Azure Stack?  Those would be slowly to deploy.

**Comment by chris-sekira** [comment]
@StephenWeatherford I just noticed that the updates went live this week. The docs were updated on the 12th https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/azure-resource-manager/templates/template-functions-object.md and I've tested that the `createObject()`, `null()`, `false()`, and `true()` are present and work (public cloud)

I came here to potentially add a separate issue. The extension now recognizes `createObject()` however it reports calling the function with no parameters as an error, which is untrue. The function can be called without parameters in order to act as a "empty object" literal essentially. Before this, you had to declare an "emptyObject" variable literal essentially.
